scripts/create_istio_system.sh

```diff
@@ -1,37 +1,49 @@
 #!/bin/bash
-istioctl version
-old_versions=$(kubectl get --ignore-not-found=true deploy istio-pilot -n istio-system -o=jsonpath='{$.spec.template.spec.containers[*].image}')
-# check if older version of istio is still installed in istio-system namespace
-if [[ $old_versions == *"1.3"* || $old_versions == *"1.4"* || $old_versions == *"1.5"* ]]; then
-    echo "Please uninstall current istio version before attempting to install $ISTIO_VERSION"
+istioctl version -i $ISTIO_NAMESPACE
+if [[ $FORCE_INSTALL == false ]]; then
+  old_versions=$(kubectl get --ignore-not-found=true deploy istio-pilot -n istio-system -o=jsonpath='{$.spec.template.spec.containers[*].image}')
```

Comment:
```suggestion
  old_versions=$(kubectl get --ignore-not-found=true deploy istio-pilot -n $ISTIO_NAMESPACE -o=jsonpath='{$.spec.template.spec.containers[*].image}')
```

Reply:
I believe old versions of istio are only able to be installed in istio-system so I want to leave that hard coded. @cbron is correct, or do we want to only check the namespace we are installing into?

Reply:
Just check istio-system

Reply:
sounds good to me!